hack/e2e-internal/e2e-up.sh

```diff
@@ -29,16 +29,40 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 
 prepare-e2e
 
+function calc_next_cidr {
```

Comment:
I would like us to go with one of the two simpler solutions proposed above instead for v1.3.  Like @colhom I'm very uncomfortable with the complex bash scripting in the original version of this PR.  Agree @madhusudancs ?